JapaneseFilter/sections/specific.txt

```diff
@@ -81,6 +81,7 @@ sweetchocolate.blogterest.net##.blockAreaHeader
 ||townnews.co.jp/entry_ad.json
 ||koko-soccer.com/assets/img/base/side_title_contents.jpg
 fu.minkabu.jp##div[class^="ad_hide"]
+||res09.bignox.com/moniqi-blog/jp-bignox-blog/2023/05/pc.gif
```

Comment:
I'm not quite sure if this is used only for ad. Even if so, `2023/05` indicates this is a recent image and we don't know how long it lasts. So please change to cosmetic
```
wikiru.jp##.menubar > a[target="_blank"] > img[width="250"][height="250"]
wikiru.jp##.menubar > a[target="_blank"] + br
```

Reply:
OK, I will change.

Reply:
There's also mobile ad `wikiru.jp###content > a[href^="https://app.adjust.com"] > img`.